.pre-commit-config.yaml

```
@@ -33,6 +33,10 @@ repos:
     rev: 5.0.4
     hooks:
       - id: flake8
+        additional_dependencies:
+        - flake8-pyi
+        types: []
+        files: ^.*.pyi?$
```

This has stopped flake8 from running on the repo’s Python files, so we need two configurations. Also, please pin the version of flake8-pyi so there are no surprise failures when a new version is released.

```suggestion
      - id: flake8
      - id: flake8
        additional_dependencies:
        - flake8-pyi==22.10.0
        types: []
        files: ^.*.pyi?$
```

---

@adamchainz the glob includes `.py` files (note the trailing question mark), so the `flake8` hook will run on both `.py` and `.pyi` files. I can introduce a second hook if you want tho 😎

---

Good point, missed that! No need for a second hook, I think, unless flake8-pyi rules will be problematic for plain python files.